website/source/guides/configuration/generate-root.html.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: "guides"
```

Comment:
this guide doesn't follow the same format as the others - did you leave out the "prerequisites" and "time to complete" sections intentionally?

Reply:
"challenge" too

Reply:
This was written by someone else, and I see that the Eng team has updated this guide due to the CLI command change.  So, I'm reverting this back to the master version.  I probably shouldn't have touched the existing guide.

Reply:
it's a good question though - should we update the existing guides to follow the new standard format?